### PR TITLE
Add production files for kubernetes

### PR DESCRIPTION
**Pull request by q3k**
We use Kubernetes via Google Container Engine to deploy the metabackend on production.

This is the preliminary commit of production files. They manage:

 - a geth instance running a rinkeby full node
 - a metabackend deployment, which connects to geth and a Google Cloud SQL PostgreSQL database
 - an ingress for metabackend (0.api.getline.in)
 - kube-lego for automatic Let's Encrypt certificates on the aforementioned ingress

### CODE REVIEW

File: production/geth/rinkeby/start.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+if [ ! -e /app/rinkeby/geth/chaindata ]; then
+	echo "Initializing rinkeby from rinkeby.json..."
+	geth --datadir=/app/rinkeby init /app/rinkeby.json
+fi
+echo "Starting geth..."
+exec geth --networkid=4 --datadir=/app/rinkeby --bootnodes=enode://a24ac7c5484ef4ed0c5eb2d36620ba4e4aa13b8c84684e1b4aab0cebea2ae45cb4d375b77eab56516d34bfbd3c1a833fc51296ff084b770b94fb9028c4d25ccf@52.169.42.101:30303 --rpc --rpcapi db,eth,net,web3,personal --rpcaddr 0.0.0.0
```
Comment (ritave) [inline production/geth/rinkeby/start.sh]: czemu nie uzyjesz po prostu --rinkeby?

File: production/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+    kubectl apply -f production/k8s/metabackend/deployment.yaml
+
+Version management
+------------------
+
+Currently version management is done manually.
+
+To build a new version of metabackend:
+    TAG=gcr.io/third-hope-184611/metabackend:$(date +%s)
```
Comment (ritave) [inline production/README.md]: dodaj nowa linie, nie formatuje sie poprawnie

File: production/geth/rinkeby/rinkeby.json [hunk 3]
```diff
@@ -0,0 +1 @@
...00000000000000000000000000f0":{"balance":"0x1"},"00000000000000000000000000000000000000f1":{"balance":"0x1"},"00000000000000000000000000000000000000f2":{"balance":"0x1"},"00000000000000000000000000000000000000f3":{"balance":"0x1"},"00000000000000000000000000000000000000f4":{"balance":"0x1"},"00000000000000000000000000000000000000f5":{"balance":"0x1"},"00000000000000000000000000000000000000f6":{"balance":"0x1"},"00000000000000000000000000000000000000f7":{"balance":"0x1"},"00000000000000000000000000000000000000f8":{"balance":"0x1"},"00000000000000000000000000000000000000f9":{"balance":"0x1"},"00000000000000000000000000000000000000fa":{"balance":"0x1"},"00000000000000000000000000000000000000fb":{"balance":"0x1"},"00000000000000000000000000000000000000fc":{"balance":"0x1"},"00000000000000000000000000000000000000fd":{"balance":"0x1"},"00000000000000000000000000000000000000fe":{"balance":"0x1"},"00000000000000000000000000000000000000ff":{"balance":"0x1"},"31b98d14007bdee637298086988a0bbd31184523":{"balance":"0x200000000000000000000000000000000000000000000000000000000000000"}},"number":"0x0","gasUsed":"0x0","parentHash":"0x0000000000000000000000000000000000000000000000000000000000000000"}
```
[the remark this answers is not in the document]
Comment (q3k) [reply production/geth/rinkeby/rinkeby.json]: This is straight from rinkeby.io, I'd rather not modify it.

File: production/geth/rinkeby/start.sh (hunk 1 above)
Comment (q3k) [reply production/geth/rinkeby/start.sh]: Just following orders from rinkeby.io.

**Pull request merged**